src/protocol/commands.rs

```diff
@@ -330,4 +330,12 @@ commands! {
     lldb_register_info {
         "qRegisterInfo" => _qRegisterInfo::qRegisterInfo,
     }
+
+    libraries use 'a {
```

Comment:
Like we discussed - let's nix the untested bit from this PR, and just stick to landing `qXfer:libraries-svr4:read` for now